modules/metering-store-data-in-shared-volumes.adoc

```diff
@@ -5,38 +5,82 @@
 [id="metering-store-data-in-shared-volumes_{context}"]
 = Storing data in shared volumes
 
+Metering has no storage by default, but it can use any ReadWriteMany PersistentVolume or any StorageClass that provisions a ReadWriteMany PersistentVolume.
+
 [NOTE]
 ====
-NFS is not recommended to use with metering.
-====
+NFS is not recommended to use in production. Using an NFS server on RHEL as storage backend can fail to meet metering requirements and to provide the performance that is needed for the Metering Operator to work appropriately.
 
-Metering has no storage by default, but it can use any ReadWriteMany PersistentVolume or any StorageClass that provisions a ReadWriteMany PersistentVolume.
+Other NFS implementations on the marketplace might not have these issues, such as a Parallel Network File System (pNFS). pNFS is an NFS implementation with distributed and parallel capability. Contact the individual NFS implementation vendor for more information on any testing that was possibly completed against {product-title} core components.
+====
 
 .Procedure
-* To use a ReadWriteMany PersistentVolume for storage, modify the `shared-storage.yaml` file below.
+You can use xref:../../storage/persistent_storage/persistent-storage-ocs.adoc#[Red Hat OpenShift Container Storage] as a ReadWriteMany PersistantVolume for the Metering Operator.
 
+.  Install Red Hat OpenShift Container Storage.
+.  Modify the `shared-storage.yaml` file to use Red Hat OpenShift Container Storage:
++
 [source,yaml]
 ----
 apiVersion: metering.openshift.io/v1
 kind: MeteringConfig
 metadata:
-  name: "operator-metering"
+  name: openshift-metering
+  namespace: openshift-metering
 spec:
   storage:
-    type: "hive"
     hive:
       type: "sharedPVC"
       sharedPVC:
-        claimName: "metering-nfs" <1>
-        # uncomment the lines below to provision a new PVC using the specified <2>
-        # storageClass.
-        # createPVC: true
-        # storageClass: "my-nfs-storage-class"
-        # size: 5Gi
+        createPVC: true
+        storageClass: "ocs-storagecluster-cephfs"
+        size: "5Gi"
+    type: hive
+  hive:
+    spec:
+      metastore:
+        storage:
+          class: "ocs-storagecluster-cephfs"
+          size: "5Gi"
 ----
 
-Select one of the configuration options below:
+.Verification
 
-<1> Set `storage.hive.sharedPVC.claimName` to the name of an existing ReadWriteMany PersistentVolumeClaim (PVC). This is necessary if you do not have dynamic volume provisioning or want to have more control over how the PersistentVolume is created.
```

Comment:
We may still want to document these options (the `createPVC` toggle and `claimName` when `createPVC` is set to false), in addition to providing a concrete example using OCS.

Reply:
> We may still want to document these options (the `createPVC` toggle and `claimName` when `createPVC` is set to false), in addition to providing a concrete example using OCS.

@timflannagan1 I added back the commented out lines. For the concrete examples using OCS, do you mean [examples similar to what Emily provided](https://gist.github.com/EmilyM1/c36082db367def141d0975491b7953ca) when were were going the NFS route still?

Reply:
@timflannagan1 @pruan-rht Bump. Are there some  additional examples that can be included in this update? Thanks!